pymola/ast.py

```diff
@@ -211,7 +211,7 @@ def __init__(self, **kwargs):
         self.max = Primary(value=None)  # type: Union[Expression, Primary, ComponentRef, Array]
         self.nominal = Primary(value=None)  # type: Union[Expression, Primary, ComponentRef, Array]
         self.value = Primary(value=None)  # type: Union[Expression, Primary, ComponentRef, Array]
-        self.fixed = False  # type: bool
+        self.fixed = Primary(value=False)  # type: Primary
```

Comment:
The way the grammar is set up, any constant value, whether float, integer, or a boolean, gets handled as a Primary. Since we handle attributes in a generic way (lines 551-552 of parser.py), we would have to make a special exception for the attribute 'fixed' if it is not of the type Primary like its cousins.